.eslintignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 dist
 lib
-storybook/public
\ No newline at end of file
+storybook/public
+website/.next
```

Comment:
This doesn't seem to be working, still seeing these files appear when I run `yarn format` 🤔

Reply:
May need to add it to `.prettierignore` as well.

Reply:
🤦

Reply:
updated